content/blog/2020/06/2020-07-07-custom-distribution-service.adoc

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Custom Distribution Service : Phase 1 Summary"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+author: sladyn98
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+
+Hello,
```

Comment:
Would be nice to have a longer intro in the blogpost

Reply:
Yes